src/lib/TextArea.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+	export let name: string;
+	export let id: string;
+</script>
+
+<textarea
+	{name}
+	{id}
+	rows="4"
```

Comment:
We can have this a default but it would be nice to allow this to be set via the props